cmd/remove.go

```diff
@@ -72,5 +73,5 @@ func clusterRemoveFromCli(ctx *cli.Context) error {
 	if err != nil {
 		return fmt.Errorf("Failed to resolve cluster file: %v", err)
 	}
-	return ClusterRemove(clusterFile)
+	return ClusterRemove(clusterFile, nil)
```

Comment:
Just to confirm that is when the code is called from cli context, the connection should be done using old way - ssh. And the customDialer will be used only when called from the kontainer-engine